docs/guides/multicloud_cluster_with_baremetal_servers.md

```diff
@@ -93,22 +93,22 @@ raw values because it allows to check their availability in the zone you want to
     ssh <user>@<baremetal_server_ip>
     ```
 
-2. Download the multicloud-init script :
+2. Download the node-agent program :
 
     ```bash
-    wget https://scwcontainermulticloud.s3.fr-par.scw.cloud/multicloud-init.sh && chmod +x multicloud-init.sh`
+    wget https://scwcontainermulticloud.s3.fr-par.scw.cloud/node-agent_linux_amd64 && chmod +x node-agent_linux_amd64
```

Comment:
No ARM?

Reply:
I didn't add a mention because the tutorial is oriented toward Elastic Metal nodes, but maybe it's a nice addition